frontend/packages/dev-console/src/components/project-access/ProjectAccess.tsx

```diff
@@ -113,10 +118,7 @@ const ProjectAccess: React.FC<ProjectAccessProps> = ({ namespace, roleBindings,
             Role Bindings
           </Link>
           . For more information, see the{' '}
-          <ExternalLink href={`${openshiftHelpBase}authentication/using-rbac.html`}>
-            role-based access control documentation
-          </ExternalLink>{' '}
```

Comment:
I don't know if the whitespace before the period was intentional or not. (We do have an icon for `ExternalLink` links.) We should double check how this looks in the UI.

cc @sg00dwin

Reply:
I checked the page and the the whitespace in not needed in this case.`</ExternalLink>{' '}.`
The external icon is positioned correctly without it.